docs/api-testing/apifctl-cicd-integration.md

```diff
@@ -287,10 +287,10 @@ Identifies the start date of the events you want to see (e.g., `-f 2021-10-21T14
 Identifies the end date of the events you want to see (e.g., `-t 2021-10-31T15:00`).
 
 ---
-### `-t <tag(s)>`
+### `-tag <tag(s)>`
```

Comment:
there must be both options -t and -tag. 
-t is used with exec and upload commands
-tag is used with run-tag command